helpers/update-site/documentation.js

```diff
@@ -169,14 +171,14 @@ const getResourcesFiles = async () => {
     const imageFiles = [];
 
     for (const resource of resources) {
-        const images = await globby([`${constants.dirs.NODE_MODULES}/{vscode-webhint,@hint/${resource}-*,@hint/configuration-all/node_modules/{vscode-webhint,@hint/${resource}-*}}/images/**/*`], { absolute: true });
+        const images = await globby([`${documentationGlobbyPattern}/images/**/*`], { absolute: true });
 
         images.forEach((image) => {
             imageFiles.push(getImageItemFromResource(image, resource));
         });
     }
 
-    const docs = await globby([`${constants.dirs.NODE_MODULES}/{vscode-webhint,@hint/{${resources.join(',')}}-*,@hint/configuration-all/node_modules/{vscode-webhint,@hint/{${resources.join(',')}}-*}}/{README.md,docs/*.md}`], { absolute: true });
+    const docs = await globby([`${documentationGlobbyPattern}/{README.md,docs/*.md}`], { absolute: true });
```

Comment:
Same question here about `hint` being included where it wasn't before.

Reply:
Same as before, `hint` is processed in another part of the code IIRC, because the structure is different.

Reply:
See https://github.com/webhintio/webhint.io/pull/831#discussion_r325832887